src/api/conceptApi.ts

```diff
@@ -30,6 +31,34 @@ interface ConceptSearchResultJson extends SearchResultJson {
   created: string;
 }
 
+async function fetchImage(imageId: string, context: Context) {
+  const imageResponse = await fetch(`/image-api/v2/images/${imageId}`, context);
+  const image = await resolveJson(imageResponse);
+  return {
+    title: image.title.title,
+    src: image.imageUrl,
+    altText: image.alttext.alttext,
+    contentType: image.contentType,
+    copyright: image.copyright,
+  };
+}
+
+async function fetchVisualElementLicense(
+  visualElement: string,
+  resource: string,
+  context: Context,
+): Promise<GQLBrightcoveLicense | GQLH5pLicense> {
+  const host = localConverter ? 'http://localhost:3100' : '';
+  const metaDataResponse = await fetch(
+    encodeURI(
+      `${host}/article-converter/json/${context.language}/meta-data?embed=${visualElement}`,
+    ),
+    context,
+  );
+  const metaData = await resolveJson(metaDataResponse);
+  return metaData.metaData[resource][0];
+}
+
```

Comment:
Kunne eventuellt skilt dette ut i egne apis. article-converter brukes også i articleApi.ts mens image-api ikkje brukes andre plasser.

Reply:
Så flytte fetchVisualElementLicense til articleApi og lage et eget imageApi for fetchImage?

Reply:
Tenkte meir på at du kunne laga articleConverterApi med denne metoden og den fra articleApi. Men det er ikkje så farlig.